lang-en/words-z.txt

```diff
@@ -250,7 +259,7 @@ zodiac
 zombie
 zombies
 zone
+zonked
 zoo
 zoology
-zoom
-zoomed
```

Comment:
Maybe it's worth keeping the word `zoomed`.

Reply:
I did not consciously remove zoom or zoomed. I'm not sure how they got removed. I would keep both.

My fork seems a bit weird. It keeps including all those previous commits above. I don't really understand why. I synced upstream first but it still seems to think they have not been committed.